mne/coreg.py

```diff
@@ -1245,3 +1248,642 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
                 F_mri_ras, 'ras', 'ras'),
             F_ras_mni, 'ras', 'mni_tal')
     _write_fs_xfm(fname_to, T_ras_mni['trans'], kind)
+
+
+def _read_surface(filename):
+    bem = dict()
+    if filename is not None and op.exists(filename):
+        if filename.endswith('.fif'):
+            bem = read_bem_surfaces(filename, verbose=False)[0]
+        else:
+            try:
+                bem = read_surface(filename, return_dict=True)[2]
+                bem['rr'] *= 1e-3
+                complete_surface_info(bem, copy=False)
+            except Exception:
+                raise ValueError(
+                    "Error loading surface from %s (see "
+                    "Terminal for details)." % filename)
+    return bem
+
+
+class Coregistration(object):
+    """Class for MRI<->head coregistration.
+
+    Parameters
+    ----------
+    info : instance of Info
+        The measurement info.
+    subject : str
+        Name of the subject the data are defined for.
+    subjects_dir : path-like
+        Path to MRI subjects directory.
+    fids : dict | str
+        List of fiducials or if set to 'auto', the fiducials are initialized
+        automatically using fiducials defined in MNI template. Defaults to
+        'auto'.
```

Comment:
@GuillaumeFavelier can you be explicit in which unit and coordinate systems the fids (which I would actually rename fiducials) should be? Should it be provided in MRI Surface RAS ? in head ? in MRI voxel?